Meadowlark-js/tests/e2e/helpers/Credentials.ts

```diff
@@ -3,74 +3,166 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-import { baseURLRequest } from '../Setup';
-
-// eslint-disable-next-line no-shadow
-export enum Clients {
-  Vendor1,
-  Vendor2,
-  Host1,
-  Assessment1,
+import memoize from 'fast-memoize';
+import { baseURLRequest } from './Shared';
+
+export type Clients = 'Vendor' | 'Host' | 'Admin';
+
+type Credentials = {
+  clientName: string;
+  roles: Array<string>;
+  key?: string;
+  secret?: string;
+  token?: string;
+};
+
+async function getAdminAccessToken(): Promise<string> {
+  let adminAccessToken = process.env.ADMIN_AT;
+  if (adminAccessToken == null) {
+    adminAccessToken = await baseURLRequest()
+      .post('/oauth/token')
```

Comment:
I might modify this in my ticket, so that it reads this from the environment variables. But the env var will have a full URL, not just an endpoint - because of the possibility of integrating a different OAuth provider. Once we get to that point, we would probably still be running e2e tests using the built-in provider. So on second thought, I'll leave this in place. Keeping my comment though just to share some thought process.